care/users/api/serializers/user.py

```diff
@@ -39,6 +42,54 @@ def create(self, validated_data):
         return super().create(validated_data)
 
 
+class UserCreateSerializer(SignUpSerializer):
+    password = serializers.CharField(required=False)
+    facilities = serializers.ListSerializer(
+        child=serializers.IntegerField(), required=False, allow_empty=True, write_only=True
+    )
+
+    class Meta:
+        model = User
+        include = ("facilities",)
+        exclude = ()
+
+    def validate_facilities(self, facility_ids):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5345345)